src/datasources/flow-ds-react/helpers.ts

```diff
@@ -945,3 +951,38 @@ export const getFilteredNodes = async ({ client, simpleRequest }, exporterNodes?
     }
 }
 
+export const convertTemplateVariables = (item: FlowQueryData, templateSrv) => {
```

Comment:
Maybe some documentation here to explain why I would use this function? The name of the function on its own is a little too vague for it to be properly self documenting.